### ISSUE TITLE
[BUG]: When no user is logged in, item previewer is invisible.

### ISSUE THREAD

**Issue by ccuser44** [AntiBoomz/BTRoblox#73]
When no there is no logged in user, item previewer is invisible.

This could be fixed when there is no player logged on the website the default character is used instead.

**Comment by DrekAnon** [comment]
feel like this is done because it checks if you own items before you can view their template for instance

**Comment by ccuser44** [comment]
> feel like this is done because it checks if you own items before you can view their template for instance

No, it just doesnt use a fallback avatar when no user is logged in

**Comment by ccuser44** [comment]
This issue still persists

**Comment by ccuser44** [comment]
Bump, still a considerable issue.
